docs/8-addendum/addendum-overview.md

```diff
@@ -0,0 +1,8 @@
+# Addendum
+
+## Section Goal
+
+This is where you will find further practice for topics covered in the bootcamp, as well as new topics not covered in the bootcamp.
+
+![goals image](../../img/goals_light.svg ':size=100x100 :class=light-mode-icon :alt= goal image; light mode')
+![goals image](../../img/goals_dark.svg ':size=100x100 :class=dark-mode-icon :alt= goal image; dark mode')
```

Comment:
Not really an addition, like all the others this was a move but github is being derpy